fe/fe-core/src/test/java/com/starrocks/clone/CloneTest.java

```diff
@@ -0,0 +1,62 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+package com.starrocks.clone;
+
+import com.starrocks.common.Config;
+import com.starrocks.pseudocluster.PseudoBackend;
+import com.starrocks.pseudocluster.PseudoCluster;
+import com.starrocks.pseudocluster.Tablet;
+import com.starrocks.server.GlobalStateMgr;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.Random;
+
+public class CloneTest {
+    @BeforeClass
+    public static void setUp() throws Exception {
+        // set some parameters to speedup test
+        Config.tablet_checker_interval_seconds = 1;
+        Config.tablet_sched_repair_delay_factor_second = 1;
+        Config.enable_new_publish_mechanism = true;
+        int fePort = new Random().nextInt(10000) + 50000;
```

Comment:
The port should not random. when testing, multiple instances will be run concurrently on one machine. It is easy to have port conflicts. it think it might use like UtFrameUtils.findValidPort()，

Reply:
ok